perllib/FixMyStreet/Cobrand/HighwaysEngland.pm

```diff
@@ -171,13 +230,37 @@ sub report_new_is_on_he_road {
         srsname => "urn:ogc:def:crs:EPSG::4326",
         typename => "Highways",
         filter => "<Filter><DWithin><PropertyName>geom</PropertyName><gml:Point><gml:coordinates>$x,$y</gml:coordinates></gml:Point><Distance units='m'>15</Distance></DWithin></Filter>",
+        accept_feature => sub { 1 },
     };
 
     my $ukc = FixMyStreet::Cobrand::UKCouncils->new;
     my $features = $ukc->_fetch_features($cfg, $x, $y);
+    my $nearest = $ukc->_nearest_feature($cfg, $x, $y, $features);
+    # National Highways responsible for litter on Motorways and AM roads
+    # And doesn't matter if we are not on a NH road
+    if ($nearest && $nearest->{properties}->{'ROA_NUMBER'} =~ /^(M|A\d+M)/) {
+        $self->{c}->stash->{report_new_is_on_he_road_not_litter} = 0;
+    } elsif (!scalar @$features) {
+        $self->{c}->stash->{report_new_is_on_he_road_not_litter} = 0;
+    }
     return scalar @$features ? 1 : 0;
 }
 
+sub _report_new_is_on_he_road_not_litter {
+    my ( $self, $x, $y ) = @_;
+
+    my $cfg = {
+        url => "https://tilma.staging.mysociety.org/mapserver/highways",
```

Comment:
Don't forget to make this point to live tilma (with the relevant layers in place) before merging.